.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 .DS_Store
 lib
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+lib/index.js
```

Comment:
lets go ahead and have this ignore everything in the `/lib` directory

`/lib/*`